newrelic/structures_newrelic_synthetics_script_api_monitor.go

```diff
@@ -78,6 +79,9 @@ func buildSyntheticsScriptAPIMonitorUpdateInput(d *schema.ResourceData) syntheti
 		if v, ok := d.GetOk("runtime_type_version"); ok {
 			input.Runtime.RuntimeTypeVersion = synthetics.SemVer(v.(string))
 		}
+	} else {
+		input.Runtime.RuntimeType = ""
+		input.Runtime.RuntimeTypeVersion = ""
```

Comment:
Test - with a created Scripted API Monitor, we can now apply to shift it to the legacy runtime and back to the new runtime too.

Uploading Screen Recording 2024-08-30 at 2.02.36 PM.mov…